### PR TITLE
contrib/certbot: new package (2.10.0)

### PR DESCRIPTION
**Pull request by a16bitsysop**
and deps python-configargparse, python-distro

### CODE REVIEW

**Comment by q66** [comment]
please split it up into batches of like max 5 templates, otherwise it's too much to take in for review

**Comment by a16bitsysop** [comment]
Okay will try and split to related batches

**Comment by a16bitsysop** [comment]
@q66 the other deps have been merged now

File: contrib/certbot/files/tmpfiles.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+d /etc/letsencrypt
+d /var/lib/letsencrypt
+d /var/log/letsencrypt 0700
```
Comment (nekopsykose) [inline contrib/certbot/files/tmpfiles.conf:3]: shouldn't certbot also have its own user and whatnot

Comment (nekopsykose) [reply contrib/certbot/files/tmpfiles.conf:3]: though maybe not, idk

Comment (a16bitsysop) [reply contrib/certbot/files/tmpfiles.conf:3]: the tmpfiles.conf is from arch linux which doesn't have a certbot user, neither does alpine.  I think in normal use it is creating certificates for things run as lots of different users.

Comment (nekopsykose) [reply contrib/certbot/files/tmpfiles.conf:3]: sure, makes sense

**Pull request merged**